instrumentation/instagocb/README.md

```diff
@@ -0,0 +1,131 @@
+# Instana instrumentation of Couchbase SDK v2 (gocb) for Go
+
+This module contains the code for instrumenting the Couchbase SDK, based on the [`gocb`](https://github.com/couchbase/gocb) library for Go.
+
+The following services are currently instrumented:
+
+1. Cluster
+   1. Query
+   2. SearchQuery
+   3. AnalyticsQuery
+2. Bucket
+3. Bucket Manager
+   1. Get/Create/Update/Drop/Flush Bucket
+4. Collection
+   1. Bulk Operations
+   2. Data operation (Insert, Upsert, Get etc)
+   3. Sub-doc operations
+      1. LookupIn & MutateIn
+5. Scope
+   1. Query
+   2. AnalyticsQuery
+6. Binary Collection
+7. Collection Manager
+8. Collection Data Structures
+   1. List
+   2. Map
+   3. Queue
+   4. Set
+
+> [!NOTE]
+> While you can call methods such as QueryIndexManager or SearchIndexManager from the provided instagocb cluster interface, it's important to note that tracing support for these methods is currently not implemented. If you find the instrumentation of any unlisted features necessary, please feel free to raise an issue.
```

Comment:
I didn't check the code yet, so maybe you have done this already: 
Would it make sense to implement these methods and just proxing to the original ones and add a debug log message saying they are not implemented? 
Or have this debug message somewhere, eg: when instacodb.Connect is called.
In any case, having this note here is already good enough, tbf.

Reply:
@willianpc : No, I haven't done this yet. 

Initially, I considered doing it, but the issue is that we might have to create all the foundational elements first before proceeding.  For example, developing an interface for QueryIndexManager and writing numerous methods for it (there's a possibility that some of these methods are not instrumented) And for these proxy methods, we may need to create unit/intergration tests as well. 

I'm thinking it might be more efficient to just add a note about this in the readme and call it good. Let me know if you disagree. I don't have any strong reasons for avoiding it, so if you still think it's necessary, we can go ahead and do it. 👍🏾